app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -116,6 +117,51 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       }
     },
 
+    populateLinkPathParts: function (clickData) {
+      var href = clickData.url
+      var path = ''
+      if (this.hrefIsRelative(href)) {
```

Comment:
This and the following `else if` result in the same outcome. Would it be possible either combine them or maybe even invert the whole statement e.g.

```
if (some condition I don't know exactly) {
  path = href.replace(/^(http:||https:)?(\/\/)([^\/]*)/, '') // eslint-disable-line no-useless-escape
} else {
  path = href
}
```

Reply:
Yes it should be `if(this.hrefIsRelative(href) || this.isMailToLink(href))`